rails-questions.md

```diff
@@ -1,30 +1,35 @@
 # ASSESSMENT 5: Database and Active Record Practical Questions
 
 1. What is SQL?
+Standard Query Language, it is the used language for relational database management, the language can update retrieve information from a data base.
 
 
-2. What the PostgreSQL query that would return all the content in a particular table?
+2. What the PostgreSQL query that would return all the content in a particular table? 
+Select *
 
 
 3. What is the command to create a new Rails application with a PostgreSQL database?
+in the command console rails new INSERT-APP-NAME -d postgresql -T
 
 
 4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
-
+in the command console rails generate model Meals breakfast:string lunch:string dinner:string
 
 5. What is a migration? Why would you use one?
-
+migration takes place when a developer update or creates new data on a table. migration is a file that is made and pushed to the schema.
```

Comment:
new column, or table, or name for the data storage. Making new data in our case would be something ORM does for us with our sql commands.